data/darktableconfig.xml.in

```diff
@@ -174,12 +174,30 @@
     <default>true</default>
     <shortdescription>do not show april 1st game</shortdescription>
   </dtconfig>
-  <dtconfig prefs="processing" section="cpugpu" restart="true">
-    <name>cache_memory</name>
-    <type factor="(1.0 / (1024.0 * 1024.0))" min="(1024 * 1024 * 100)">int64</type>
-    <default>(1024 * 1024 * 512)</default>
-    <shortdescription>memory in MB to use for thumbnail cache</shortdescription>
-    <longdescription>this controls how much memory is going to be used for thumbnails and other buffers (needs a restart).</longdescription>
+<!-- be sure to keep the code in sync when changing this enum, see common/darktable.c -->
+  <dtconfig prefs="processing" section="cpugpu">
+    <name>resourcelevel</name>
+    <type>
+      <enum>
+        <option>default</option>
+        <option>mini (debug)</option>
+        <option>reference (debug)</option>
+        <option>small</option>
+        <option>medium</option>
+        <option>large</option>
+        <option>unrestricted</option>
+      </enum>
+    </type>
+    <default>default</default>
+    <shortdescription>darktable resources</shortdescription>
+    <longdescription>defines how much darktable may take from your system resources. can be changed at runtime\n   mini: 0.5 GB free ram and 0.25GB free video ram.\n   reference: 8GB free ram and 2GB free video ram\n   unrestricted: will take almost all and might lead to swapping.</longdescription>
```

Comment:
We probably want to describe a bit more each level?

Reply:
ok, will do that

Reply:
done